index.html

```diff
@@ -1,8 +1,7 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
```

Comment:
```suggestion
<html lang="ko">
```
`lang` 속성도 바꿔주면 좋을 것 같네요🙃